web_app_drawer/static/src/css/web_app_drawer.css

```diff
@@ -0,0 +1,172 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+body {
+    width: 100%;
+    height: 100%;
+}
+
+main {
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+}
+
+/* Menu Bar */
+.css-main-nav {
+    margin-bottom: 0;
+    min-height: 2em;
+}
+
+.css-main-nav ul.nav > li > a {
+    padding: 0.625em; 
+    height: 2.5em;
+}
+
+.css-main-nav div.progress {
+    margin-top: .625em;
+}
+
+.oe_topbar_avatar {
+    height: 1.8em;
+    border-radius: 50%;
+}
+
+@media (max-width: 767px) {
+    #odooMenuBarNav[aria-expanded="false"] {
+        /* Hack to hide the visibly expanded mobile menu on load. */
+        position: absolute;
+        z-index: -9999;
+    }
+}
+
+/* Dropdown Submenu */
+
+.dropdown-submenu {
+    position: relative;
+}
+
+.dropdown-submenu > .dropdown-menu {
+    top: 0;
+    left: 100%;
+    margin-top: -0.375em;
+    margin-left: -0.0625em;
+    -webkit-border-radius: 0 0.375em 0.375em 0.375em;
+    -moz-border-radius: 0 0.375em 0.375em 0.375em;
+    border-radius: 0 0.375em 0.375em 0.375em;
+}
+
+.dropdown-submenu:hover > .dropdown-menu {
+    display: block;
+}
+
+.dropdown-submenu > a:after {
+    display: block;
+    content: " ";
+    float: right;
+    width: 0;
+    height: 0;
+    border-color: transparent;
+    border-style: solid;
+    border-width: 0.3125em 0 0.3125em 0.3125em;
+    border-left-color: #cccccc;
+    margin-top: 0.3125em;
+    margin-right: -0.625em;
```

Comment:
Most of these should be boostrap variables, but I'll stop saying that until you move to LESS :laughing: